kani-compiler/src/codegen_cprover_gotoc/context/goto_ctx.rs

```diff
@@ -390,9 +390,9 @@ fn machine_model_from_session(sess: &Session) -> MachineModel {
     let word_size = 32;
     let rounding_mode = RoundingMode::ToNearest;
 
-    MachineModel::new(
+    MachineModel {
+        architecture: architecture.to_string(),
```

Comment:
Why the reorder these? (Also, clone instead of to_string? Same thing ofc, but maybe clearer.)

Reply:
Old OCaml/JS habit of putting all the non-shorthand fields first, and then the short-hand fields. So when reading the assignment, you see what's not trivial first, in case your eyes naturally stop reading the bunch of shorthand assignments that follow

Also, here, to_string doesn't work, because it's already a string. `mm.architecture()` used to return a reference to the string, but now I'm accessing the string itself directly

Reply:
Ah, I see. Makes sense.